fixtures/dangerjs/github.example

```diff
@@ -1,4 +1,4 @@
-// @flow
+
```

Comment:
Why do some files start with an empty line?

Reply:
Thats a good question.  It took me ages to figure out why some unit tests were failing.  Turns out the line numbers are hard-coded in them and I was trying to change the minimal amount of code, so that this PR was readable.